content/lessons/chapter-3/tabconf-clue-1.tsx

```diff
@@ -0,0 +1,53 @@
+'use client'
+
+import { useProceed, useTranslations } from 'hooks'
+import { Button } from 'shared'
+import { CodeExample, Introduction, Text } from 'ui'
+
+export const metadata = {
+  title: 'chapter_three.tabconf_clue_one.title',
+  navigation_title: 'chapter_three.tabconf_clue_one.nav_title',
+  image: '/assets/images/tabconf-clue-1.webp',
+  key: 'CH3TCC1',
+}
+
+export default function BookACall1({ lang }) {
+  const t = useTranslations(lang)
+  const proceed = useProceed()
+  return (
+    <Introduction
+      lang={lang}
+      imagePosition="object-[50%_100%] md:object-top"
+      noBtn
+    >
+      <Text className="mt-4 text-lg md:text-xl">
+        {t('chapter_three.tabconf_clue_one.paragraph_one')}
+      </Text>
+      <Text className="mt-4 text-lg md:text-xl">
+        {t('chapter_three.tabconf_clue_one.paragraph_two')}
+      </Text>
+      <Text className="mt-4 text-lg md:text-xl">
+        {t('chapter_three.tabconf_clue_one.paragraph_three')}
+      </Text>
+      <Text className="mt-4 text-lg md:text-xl">
+        {t('chapter_three.tabconf_clue_one.paragraph_four')}
+      </Text>
+      <CodeExample
+        className="mt-4 text-wrap font-space-mono text-[#F6A40A]"
+        code={`${process.env.NEXT_PUBLIC_TABCONF_KEY}`}
```

Comment:
You are the best for making this an environment variable! <3 I'm talking to the TABConf guy and he might want to change it. I'll keep you posted.